generate-stackbrew-library.sh

```diff
@@ -3,12 +3,12 @@ set -Eeuo pipefail
 
 declare -A aliases=(
 	[2.6]='2 latest'
-	[2.7-rc]='rc'
+	[2.7]='2 latest'
```

Comment:
I don't think you want both 2.6 and 2.7 aliased to `2 latest`. I'm guessing you can remove the 2.6 alias. 🙂